bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -129,17 +135,31 @@
 				<option value="InputHDMI4">Input HDMI4</option>
 				<option value="InputAV1">Input AV1</option>
 				<option value="PowerOff">Power Off</option>
-				<option value="PowerOn">Power On</option>
+				<option value="POWERON">Power On</option>
 			</options>
 		</state>
 	</channel-type>
 
+	<channel-type id="control">
+		<item-type>Player</item-type>
+		<label>Control</label>
+		<description>Control playback e.g. Play/Pause/Next/Previous</description>
+		<category>Player</category>
+	</channel-type>
+
 	<channel-type id="activeApp">
 		<item-type>String</item-type>
 		<label>Active App</label>
 		<description>The Currently Running App on the Roku</description>
 	</channel-type>
 
+	<channel-type id="activeAppName">
+		<item-type>String</item-type>
+		<label>Active App Name</label>
+		<description>The Currently Running App on the Roku</description>
```

Comment:
```suggestion
		<description>The currently running app on the Roku</description>
```

Reply:
like handleCommand above, I kept this in line with the other channel-types and how they were cased.